src/clone.rs

```diff
@@ -360,6 +362,15 @@ macro_rules! clone {
             }
         }
     );
+    ($($(@ $strength:ident)? $($variables:ident).+ $(as $rename:ident)?),+ => @default-panic, move || $body:expr ) => (
```

Comment:
Couldn't this just call itself and put `{}` around `$body`? Less code duplication then